features/step_definitions/stepdefs.js

```diff
@@ -337,3 +337,46 @@ Then("my box is still ticked", async function() {
   boxSelected = await declaration1.isSelected();
   assert.equal(boxSelected, true, "box isn't selected");
 });
+
+///////// REGISTRATION SUMMARY /////////
+
+Given("I am on the registration summary page", async () => {
+  await driver.get(`http://${process.env.URLBASE}${process.env.SUMMARY}`);
+});
+
+Given("I have filled out the data in the previous sections", async () => {
+  await driver.get(`http://${process.env.URLBASE}${process.env.ESTABLISHMENTTRADINGNAMEPAGE}`);
+  establishmentTradingName = await driver.findElement(
+    webdriver.By.name("establishment_trading_name")
+  );
+  await establishmentTradingName.sendKeys(validTradingName);
+  saveContinueButton = await driver.findElement(webdriver.By.className("css-nyvlzd"));
+  await saveContinueButton.submit();
+});
+
+Then("I am taken to the declaration page", async () => {
+  currentUrl = await driver.getCurrentUrl();
+  assert.notEqual(
```

Comment:
Why are we checking for `notEqual`? (most likely just my lack of understanding)

Reply:
Error on my part, fixing now